arenadata/Dockerfile.ubuntu

```diff
@@ -42,14 +43,20 @@ ENV OUTPUT_ARTIFACT_DIR=bin_gpdb
 ENV CONFIGURE_FLAGS="--enable-debug-extensions --with-gssapi --enable-cassert --enable-debug --enable-depend" 
 
 # Compile with running mocking tests
-RUN bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash
+RUN set -eux; \
+    export PYTHON=/usr/bin/python2; \
+    bash /home/gpadmin/gpdb_src/concourse/scripts/compile_gpdb.bash; \
+    make -C gpdb_src/src/pl/plpython clean; \
```

Comment:
Is this line necessary?

Reply:
Yes, it allows to build plpython3.so with references to the python3. Otherwise plpython3.so builds with references to python2 and becomes broken.